packages/fxa-admin-panel/src/components/AccountSearch/Subscription/index.test.tsx

```diff
@@ -26,7 +26,9 @@ it('renders each field as expected', () => {
 
   screen.getByText(subscription.productName);
   screen.getByText(subscription.status);
-  expect(screen.getAllByText('1970-01-19 @', { exact: false })).toHaveLength(4);
+  expect(screen.getAllByText(/1970-01-1[89] @/, { exact: false })).toHaveLength(
```

Comment:
This was tricky. The test was failing and I thought I was going crazy cause clearly it passed at some point. Turns out the issue is that dateFormat is taking the users timezone into account. So depending on where you are you might get a different result. For example:

```
> const dateFormat = require('dateformat')
undefined
> dateFormat(new Date(1583259953), 'UTC:yyyy-mm-dd @ HH:MM:ss Z') 
'1970-01-19 @ 07:47:39 UTC'
> dateFormat(new Date(1583259953), 'yyyy-mm-dd @ HH:MM:ss Z') 
'1970-01-18 @ 23:47:39 PST'
```

In a PST timezone, 1583259953, would have renders with 1970-01-18 date and the subsequent test would pass. This makes the check a bit more flexible. Alternatively we could render everything in UTC. Not sure what's best.

Reply:
I've never used `dateFormat`. `moment` is used in `fxa-auth-server` though (and in `fxa-shared`), could be worth looking into/switching as an alternative?

Reply:
Agree that'd be nice to consistent about this. @LZoog any thoughts here?

Reply:
If we're only using `dateFormat` in the admin-panel, agreed, we could follow a follow up to replace it with `moment`; less dependencies WFM.